document/js-api/index.bs

```diff
@@ -1240,6 +1240,8 @@ This document defines a host environment for WebAssembly. It enables a WebAssemb
 
 <h2 id="esm-integration">Integration with ECMAScript modules</h2>
 
+Note: It is possible to implement the Wasm-ESM integration in two stages. In the first stage only source phase imports of Wasm are supported (`import source xModule from "./x.wasm"`). In the second stage, evaluation phase imports would be supported too (`import xInstance from "./x.wasm"`). If initially implementing just source phase imports, the `GetExportedNames`, `ResolveExport`, `InitializeEnvironment`, and `ExecuteModule` abstract operations can be implemented as abstract operations unconditionally throwing a `SyntaxError` exception. In this case, module fetch and CSP integration is still required to be implemented as specified in this proposal. Implementers are encouraged to ship both stages at once, but it is deemed for implementers to initially ship the first stage and then quickly follow up with the second stage, if this aids "time to ship" in implementations.
```

Comment:
Note, we don't actually mention CSP anywhere in this proposal. Given that, I would have interepreted this note to mean that this proposal requires `unsafe-wasm-eval`. I would suggest either removing the CSP point, or adding a new CSP section where we discuss CSP policy integration.